.ci/Jenkinsfile

```diff
@@ -391,23 +391,6 @@ stage('Build') {
                 }
             }]
         }
-        else if (!isComposerImage) {
```

Comment:
Would it make sense to do this in it's own PR?  Just thinking if anything goes wrong it would be easier to revert a specific commit.